package.json

```diff
@@ -29,6 +29,7 @@
   "dependencies": {
     "@discordjs/builders": "^0.12.0",
     "@discordjs/rest": "^0.3.0",
+    "@types/node": "*",
```

Comment:
This should be a dev dependency. 
It should also have a version specifier, any particular reason you have used `*` instead ?

Reply:
I used what showed in the warning which was `@types/node/*`.